main.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env pybricks-micropython
-
```

Comment:
The blank line wasn't causing the issue. I assume you were seeing the following error.
```
/usr/bin/env: 'pybricks-micropython\r': No such file or directory
----------
Exited with error code 127.
```

The fix for this is correcting the carriage return value within VS Code. You need to select `LF` (please see the screenshot). If `CRLF` is chosen, you will see the error mentioned above. 
![image](https://user-images.githubusercontent.com/60576292/91246564-c09c0000-e715-11ea-80fa-2b7e1d8658ed.png)

Can we leave the blank line as it was, so it matches with other files? Other than that, everything looks good. 👍

Reply:
Fixed it!